src/pyscaffold/templates/setup_cfg.template

```diff
@@ -32,16 +32,16 @@ classifiers =
 
 [options]
 zip_safe = False
-packages = find:
+packages = find_namespace:
```

Comment:
Alternatively we could choose dynamically between `find` and `find_namespace` depending on the `namespace` being activated or not...

Keeping everything with `find_namespace` is easier... and I don't think we have a drawback since we use the `src` layout (in the docs it is explicitly said `find_namespace_package` is backwards compatible with `find_package`).

Reply:
If `find_namespace:` works I would also go for the simple solution since choosing dynamically just introduces another source of possible errors and needs more checks. I like simple solutions :-)